# request 
---
# response
bool light_state